### ISSUE TITLE
New Segment: Azure PowerShell

### ISSUE THREAD

**Issue by sassdawe** [JanDeDobbeleer/oh-my-posh#5110]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What would you like to see added?

I understand that the Azure segment currently supports both the Azure CLI and the Azure PowerShell module, but I think it would be awesome if oh-my-posh could show when the two are connected to different tenants/subscriptions in the same time. 

Hence the request to have a dedicated segment for AzPowerShell and AzPowerShell only.

Basically have two segments showing the connection information in the same time could warn people when the context of the two tools don't match, and help the users realize they might forgot to connect/reconnect before they execute something they shouldn't against the wrong target.

**Comment by JanDeDobbeleer** [comment]
@sassdawe you can have two `az` segments with a different source which results in the same, no?

**Comment by sassdawe** [comment]
I tried. **Assuming** I did the segment configuration correctly - by adding `source. 

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/10754765/77ff0a85-7eab-40ab-b2ae-41e1ee20dc0a)

It seems that I just see the CLI's context twice even when AzPowerShell is not connected.

**Comment by JanDeDobbeleer** [comment]
@sassdawe you added the properties in the wrong location, so it defaults to the CLI:

```json
{
  "type": "az",
  "style": "powerline",
  "powerline_symbol": "",
  "foreground": "#000000",
  "background": "#9ec3f0",
  "template": "  {{ .EnvironmentName }}",
  "properties": {
    "source": "cli"
  }
},
{
  "type": "az",
  "style": "powerline",
  "powerline_symbol": "",
  "foreground": "#000000",
  "background": "#9ec3f0",
  "template": "  {{ .EnvironmentName }}",
  "properties": {
    "source": "pwsh"
  }
}
```

Make sure you have `$env:POSH_AZURE_ENABLED = $true` in your `$PROFILE` after initialising Oh My Posh to enable the PowerShell module AND have the module imported.

**Comment by sassdawe** [comment]
Thanks @JanDeDobbeleer !

Can I submit a PR to update the sample with the usage of properties for the Azure segment, would I need an issue for that?

**Comment by JanDeDobbeleer** [comment]
You can update the segment docs by adding the `properties`, yes.
